pkg/controllers/resources/nodes/translate.go

```diff
@@ -20,6 +19,8 @@ var (
 	TaintsAnnotation = "vcluster.loft.sh/original-taints"
 )
 
+const ResourcePods = "pods"
```

Comment:
nit-pick: `corev1.ResourcePods` can be used instead of defining this constant here